stripe-core/src/main/java/com/stripe/android/core/networking/StripeRequest.kt

```diff
@@ -11,12 +11,12 @@ abstract class StripeRequest {
     /**
      * The type(Get/Post/Delete) of this request.
      */
-    internal abstract val method: Method
+    abstract val method: Method
```

Comment:
RestrictTo?

Reply:
`RestritTo` can't be allied to abstract fields, I think it's fine since the class itself is already annotated